mastering/custom_cache.rst

```diff
@@ -3,14 +3,14 @@
 Conan local cache: concurrency, Continuous Integration, isolation
 =================================================================
 
-Conan needs access to some, per user, configuration files, as the **conan.conf** file that
+Conan needs access to some per user configuration files, such as the **conan.conf** file that
 defines the basic client app configuration. By convention, this file will be located in the
-user home folder **~/.conan/**. This folder will typically also store the package cache, in
-**~/.conan/data**. Though the latter is configurable in *conan.conf*, conan needs
+user home folder **~/.conan/**. This folder will also typically store the package cache in
+**~/.conan/data**. Even though the latter is configurable in *conan.conf*, cConan needs
```

Comment:
minor ``cConan`` issue here

Reply:
Good catch.

Reply:
I believe I've corrected it. Please review again.